### PR TITLE
Implement WorkFromClient

### PR DESCRIPTION
**Pull request by pepibumur**
### Short description
Implement the client that exposes the requests as Observables and passes the appId to all the requests.

### Implementation

- [x] WorkFromClient
- [x] WorkFromClient Endpoints
- [x] Tests

### GIF

gif

### CODE REVIEW

**Comment by pepibumur** [comment]
Review @sergigracia / @isaacroldan

File: Example/Tests/Client/WorkFromClientPlacesSpec.swift [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+            it("should use the correct request") {
+                waitUntil(action: { (done) in
+                    _ = subject.placeIndex(latitude: 33, longitude: 33, dayOfTheWeek: 3, radius: 3).subscribeCompleted{
+                        expect(subject.requested) == subject.requestFactory.placeIndex(latitude: 33, longitude: 33, appId: "333", dayOfTheWeek: 3, radius: 3)
+                        done()
+                    }
+                })
+            }
+        }
+
```
Comment (sergigracia) [inline Example/Tests/Client/WorkFromClientPlacesSpec.swift]: 🎨  ?

File: Example/Tests/Client/WorkFromClientPlacesSpec.swift [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+                        done()
+                    }
+                })
+            }
+        }
+        
+    }
+}
+
+
```
Comment (sergigracia) [inline Example/Tests/Client/WorkFromClientPlacesSpec.swift]: 🎨  ?

File: Example/Tests/Client/WorkFromClientPlacesSpec.swift [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    init(response: JSON) {
+        self.response = response
+        super.init(appId: "333", requestFactory: RequestFactory())
+    }
+    
+    private override func request(request: NSURLRequest) -> Observable<(JSON, NSURLResponse?)> {
+        requested = request
+        return Observable.just((response, nil))
+    }
+
```
Comment (sergigracia) [inline Example/Tests/Client/WorkFromClientPlacesSpec.swift]: empty lines at the end of a class are intentionally done?

**Comment by sergigracia** [comment]
screen shot 2016-09-06 at 20 09 36

**Comment by sergigracia** [comment]
👍  Good job @pepibumur 👏

**Comment by pepibumur** [comment]
@sergigracia I don't use `swiftlint` for tests. I think in this case we should go ahead without linting. What do you think?

**Comment by pepibumur** [comment]
------
When a client asks for an IE compatibility
--
alt text

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Comment by sergigracia** [comment]
Agree @pepibumur. Let's 🏃

**Pull request merged**